code/modules/organs/limbs.dm

```diff
@@ -756,8 +756,7 @@ Note that amputating the affected organ does in fact remove the infection from t
 	if(body_part == CHEST)
 		return FALSE
 
-	remove_limb_flags()
-	add_limb_flags(LIMB_BLEEDING | LIMB_DESTROYED)
+	set_limb_flags(LIMB_DESTROYED)
```

Comment:
Cant this override robotic limbs?

Reply:
It'll clear out the limb_robotic flag, yeah, but that's fine because none of the behavior specific to robotic limbs matters when it's missing (reduced damage taken, no bleeding, EMP, different drop text/repairs), the only way to fix a missing limb is via robotic replacement anyway, and I think it's basically reasonable to say a limb you don't have isn't robotic.